webui/react/src/components/FilterForm/components/FilterField.tsx

```diff
@@ -172,6 +173,19 @@ const FilterField = ({
     [field.type, formStore, index, inputOpen, isSpecialColumn, parentId],
   );
 
+  const selectedUser =
+    field.columnName === 'user'
+      ? Loadable.getOrElse(null, userStore.getUser(Number(field.value)).get())
```

Comment:
You can't use `.get` here, you need to correctly hook on the observable.

Reply:
`.get` can only be used inside _actions_ (and only when there is no possibility of racing), you can never use it in the render function.